sections/semantics-embedded-content.include

```diff
@@ -2164,6 +2167,9 @@
   The <dfn attribute for="HTMLImageElement"><code>isMap</code></dfn> IDL attribute must <a>reflect</a>
   the <code>ismap</code> content attribute.
 
+  The <dfn attribute for="HTMLImageElement"><{img/longDesc}> IDL attribute must <a>reflect</a>
```

Comment:
Does that build? If so, does it link to the right place? Hmmm. Usually inside of a dfn element, we only use `<code>` rather than attribute refs. See `src` and `srcset` above for examples. (And you want to make an IDL ref anyway which is `{{HTMLImageElement/longDesc}}` rather than what you've got here.)